commands.go

```diff
@@ -3,475 +3,289 @@ package main
-		rest := strings.TrimSpace(strings.TrimPrefix(line, "./color"))
-		if rest == "which" {
-			b(devbot, "you're using "+u.color)
-			return
-		}
-		if err := u.changeColor(rest); err != nil {
-			b(devbot, err.Error())
-		}
-		return
+	if len(args) > 1 {
+		target.ban(u, strings.Join(args[1:], " "))
+	} else {
+		target.ban(u, "")
 	}
-	if line == "./people" {
-		b("", `
-**Hack Club members**  
-Zach Latta     - Founder of Hack Club  
-Zachary Fogg   - Hack Club Game Designer  
-Matthew        - Hack Club HQ  
-Caleb Denio, Safin Singh, Eleeza A  
-Jubril, Sarthak Mohanty, Anghe,  
-Tommy Pujol, Sam Poder, Rishi Kothari,  
-Amogh Chaubey, Ella Xu, Hugo Hu,  
-Robert Goll, Tanishq Soni, Arash Nur Iman,  
-Temi, Aiden Bai, Ivan Bowman, @epic  
-_Possibly more people_
-
-
-**From my school:**  
-Kiyan, Riya, Georgie  
-Rayed Hamayun, Aarush Kumar
-
-
-**From Twitter:**  
-Ayush Pathak    @ayshptk  
-Bereket         @heybereket  
-Sanketh         @SankethYS  
-Tony Dinh       @tdinh\_me  
-Srushti         @srushtiuniverse  
-Surjith         @surjithctly  
-Arav Narula     @HeyArav  
-Krish Nerkar    @krishnerkar\_  
-Amrit           @astro_shenava  
-Mudrank Gupta   @mudrankgupta  
-Harsh           @harshb__
-
-**And many more have joined!**`)
+	u.system("Banned!")
+}
+
+func idCommand(u *user, args []string) {
+	if len(args) != 1 {
+		u.system(u.id)
 		return
 	}
 
-	if line == "./help" || line == "cat README.md" {
-		b("", `Welcome to Devzat! Devzat is chat over SSH: github.com/quackduck/devzat  
-Because there's SSH apps on all platforms, even on mobile, you can join from anywhere.
 
-Interesting features:
-* Many, many commands. Run ./commands.
-* Rooms! Run ./room to see all rooms and use ./room #foo to join a new room.
-* Markdown support! Tables, headers, italics and everything. Just use \\n in place of newlines.
-* Code syntax highlighting. Use Markdown fences to send code. Run ./example-code to see an example.
-* Direct messages! Send a quick DM using =user <msg> or stay in DMs by running ./room @user.
-* Timezone support, use ./tz Continent/City to set your timezone.
-* Built in Tic Tac Toe and Hangman! Run ./tic or ./hang <word> to start new games.
-* Emoji replacements! \:rocket\: => :rocket: (like on Slack and Discord)
-
-For replacing newlines, I often use bulkseotools.com/add-remove-line-breaks.php.
-
-Made by Ishan Goel with feature ideas from friends.  
-Thanks to Caleb Denio for lending his server!`)
+	target, ok := findUserByName(u.room, args[0])
+	if !ok {
+		u.system("User not found!")
 		return
 	}
-	if strings.HasPrefix(line, "cat") {
-		if line == "cat" {
-			b("", "usage: cat [-benstuv] [file ...]")
+	u.system(target.id)
+}
+
+func helpCommand(u *user, args []string) {
+	u.system("**Commands**")
+	for _, command := range commands {
+		if command.requiresAdmin {
+			if auth(u) {
+				u.system(fmt.Sprintf("%s - %s %s", green.Paint(command.name), command.description, red.Paint("(ADMIN ONLY)")))
```

Comment:
I would really like it if you would send out the entire message all at once, instead of many little ones (kinda spammy)

This does not need to be from "SYSTEM"

Reply:
This just makes the code cleaner and you dont really notice it

Reply:
Actually, since this will be broadcasted and thus added to the backlog, it makes a significant difference, it's a smol change

Reply:
yeah uh backlog is to know what people are talking about, not the output of commands.

Reply:
Fine, let's keep command output private.